Updated OwOCord... again

hetvos opened the pull request:
1. Changed all `replace`'s to `replaceAll`'s for faster owoify-ing.
2. Set interval back to 250ms due to change 1.

hetvos committed:
Updated OwOCord... again

1. Changed all `replace`'s to `replaceAll`'s for faster owoify-ing.
2. Set interval back to 250ms due to change 1.

CanadaHonk commented on modules/fun/owoCord.js:
```diff
@@ -1,13 +1,13 @@
 		let messages = document.getElementsByClassName("messageContent-2qWWxC");
 		for(let message of messages) {
-			message.textContent = message.textContent.replace("r","w").replace("l","w").replace("R","W").replace("L","W").replace("(edited)"," [Edited]");
+			message.textContent = message.textContent.replaceAll("r","w").replaceAll("l","w").replaceAll("R","W").replaceAll("L","W").replaceAll("(edited)"," [Edited]");
```
`String.prototype.replaceAll` is still pretty new and I'm unsure whether Discord Stable is up to date enough to support it.

hetvos replied:
alright, i'll change it later to another method that is compatible for sure

CanadaHonk replied:
You can probably use Regex like so:
```javascript
'foobar'.replace(/o/g, 'e')
```

Fjorge0 replied:
my (slightly unrelated) suggestion is to use
```js
'foobar'.replace(/(?<=o).?(?=o)/g, 'w')
```
for better potential owo detection
what this will do (afaik) is replace any character in between two o's with a w

hetvos replied:
@FjorgeTheHuman that's not what OwOCord does though.
@Oj18 updated the code to be compatible with Discord Stable for sure

hetvos committed:
Updated OwOCord for the 3rd time

**Changelog:**
* Updated code to be compatible with Discord Stable for sure.
* Separated conversion to "OwOSpeak" into a custom string method (owocordToOwOSpeak)

CanadaHonk commented on modules/fun/owoCord.js:
```diff
@@ -1,13 +1,17 @@
 
 let owointerval;
 function owoify() {
+	String.prototype.owocordToOwOSpeak = function() {
```
Could you not pollute JS prototypes like this and just use a normal function please? Thanks.

hetvos replied:
Alright

hetvos replied:
done

hetvos committed:
Updated OwOCord for the... uh... 4th time

`owocordToOwOSpeak` is no longer a string method

CanadaHonk commented on modules/fun/owoCord.js:
```diff
@@ -16,6 +20,7 @@ let obj = {
 	},
 	remove: async function () {
 		clearInterval(owointerval);
+		String.prototype.owocordToOwOSpeak = undefined;
```
Forgot to remove this?

hetvos replied:
yes because im dum, removing it now

hetvos replied:
removed it

hetvos committed:
5th time's the charm! Wait it's supposed to be 3rd?

Removed leftover code from when i used a string method which i did not remove before because im dumb and forgot about it lmfao
